modules/nf-core/phispy/main.nf

```diff
@@ -0,0 +1,53 @@
+process PHISPY {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::phispy=4.2.21"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/phispy:4.2.21--py310h30d9df9_1':
+        'biocontainers/phispy:4.2.21--py310h30d9df9_1' }"
+
+    input:
+    tuple val(meta), path(gbk)
+
+    output:
+    tuple val(meta), path("${prefix}/")              , emit: results
```

Comment:
Do you need the whole directory as input for another module? Otherwise I would suggest emitting all files separately

Reply:
My thinking here is that, since PhiSpy can generate additional output files ([Reference](https://github.com/linsalrob/PhiSpy#output-files)), if someone using the module is generating these optional files via task.ext.args, they can choose to access them by using the results channel.

Reply:
I'm doing something similar in a workflow with the [panaroo/run module](https://github.com/nf-core/modules/blob/master/modules/nf-core/panaroo/run/main.nf) for instance

Reply:
> > I'm doing something similar in a workflow with the [panaroo/run module](https://github.com/nf-core/modules/blob/master/modules/nf-core/panaroo/run/main.nf) for instance
> 
> I get what you mean. @nf-core/maintainers anyone got an opinion on this?

I agree with you, @nvnieuwk. Using optional output-channels seems to be the standard for something like this.

Reply:
Exporting a whole directory can be OK IMO when there is a huge number of files and nested subdirectories.

Normally in such cases I would export the directory, _and_ additional dedicated channels for specific files too.

So I agree to both having the folder and the additional optional ones :+1:

Reply:
Thank you for the suggestions, I've added optional output-channels.